scripts/install

```diff
@@ -22,6 +22,10 @@ else
       source ./lib/install_kafka.sh
       install_kafka
       ;;
+    nginx-rtmp)
+      source ./lib/install_nginx_rtmp.sh
+      install_nginx_with_rtmp
```

Comment:
Don't you also want to add these two lines to the if statement above?

Reply:
Done